platform_channels/pubspec.yaml

```diff
@@ -4,19 +4,19 @@ description: A new Flutter project.
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: ">=2.12.0-0 <3.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
 
-  cupertino_icons: ^1.0.0
+  cupertino_icons: ^1.0.2
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
 
-  pedantic: ^1.9.0
+  pedantic: ^1.10.0
```

Comment:
nit: mark as `pedantic: ^1.11.0`

Reply:
> nit: mark as `pedantic: ^1.11.0`

Sure @domesticmouse I'll do the change :)